src/antsibull_nox/paths.py

```diff
@@ -175,8 +177,23 @@ def copy_collection(source: Path, destination: Path) -> None:
     copier.copy(source, destination, exclude_root=[".nox", ".tox"])
 
 
+def create_temp_directory(basename: str) -> Path:
+    """
+    Create a temporary directory outside the nox tree.
+    """
+    directory = tempfile.mkdtemp(prefix=basename)
```

Comment:
Does this actually work? I thought nox sets $TMPDIR for the whole process.

Reply:
For me it did, the directory ended up in `/tmp`.